bindings/python/src/cloudsmith_api/models/package_dependencies.py

```diff
@@ -33,7 +33,7 @@ class PackageDependencies(object):
                             and the value is json key in definition.
     """
     swagger_types = {
-        'dependencies': 'list[PackageDependency]'
+        'dependencies': 'str'
```

Comment:
Is this kosher?

Reply:
I realise this wasn't what you were working on, but... might need looked into I'm afraid.

Reply:
No idea why its done this, noticed a similar issue with the go bindings but a fix to the main app fixed them...